.github/workflows/tests.yml

```diff
@@ -147,7 +151,7 @@ jobs:
         with:
           push: true
           file: dockerfiles/semgrep-dev.Dockerfile
-          platforms: ${{ matrix.platform }}
+          platforms: linux/amd64,linux/arm64
```

Comment:
I'm not sure if this means anything internally to maintainers, but from what I understand the current `{{ matrix.platform }}` usage here would result to empty string because I don't see any `strategy.matrix.platform` block. Let me know if there is a more desirable entry here.